### PR TITLE
Add activations to paddle.nn.functional.activation

### PR DESCRIPTION
**Pull request by HaiderSultanArc**

### CODE REVIEW

**Comment by HaiderSultanArc** [comment]
Added elu, leaky_relu, relu, sigmoid, silu, softmax, softplus, swish, mish, tanhshrink and thresholded_relu activation functions to paddle.nn.functional.activation
Issue: #15050
